lib/capistrano/slack_notify.rb

```diff
@@ -91,8 +91,12 @@ def destination
       fetch(:slack_destination, stage)
     end
 
+    def repository
+      fetch(:repository, 'origin')
```

Comment:
`origin` is a remote, not a repository. what's your reasoning for using `:repository` here?